domain/src/main/java/org/oppia/android/domain/state/BUILD.bazel

```diff
@@ -0,0 +1,44 @@
+"""
+Domain services & definitions corresponding to managing ephemeral states in play sessions.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "state_deck",
+    srcs = [
+        "StateDeck.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        "//model:exploration_checkpoint_java_proto_lite",
+        "//model:exploration_java_proto_lite",
+        "//model:subtitled_html_java_proto_lite",
+    ],
+)
+
+kt_android_library(
+    name = "state_graph",
+    srcs = [
+        "StateGraph.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        "//model:exploration_checkpoint_java_proto_lite",
+        "//model:exploration_java_proto_lite",
+    ],
+)
+
+kt_android_library(
+    name = "state_list",
+    srcs = [
+        "StateList.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        "//model:event_logger_java_proto_lite",
+        "//model:exploration_checkpoint_java_proto_lite",
```

Comment:
Are these two needed?

Reply:
Yes I think it is needed

Reply:
import org.oppia.android.app.model.Outcome
because of this import on line 4 in StateList.kt

Reply:
import org.oppia.android.app.model.State
on line 6 in StateList.kt

Reply:
We should be using exploration_java_proto_lite, then, for those (not the checkpoint version of it).